examples/cars/configs/train_cars.yaml

```diff
@@ -38,6 +38,7 @@ metric_args:
   cmc_top_k: [1]
   map_top_k: [5]
   fmr_vals: [1]
+  explained_variance_ths: [0.5, 0.9, 0.99]
```

Comment:
@dapladoc my idea was to use `pfc_variances` because `pfc_explained_variance_ths` is too long as I mentioned before. Anyway, pfc is our own metric, so, there is no chance for the argument to be self-explanatory, so, let's keep the short version: `pfc_variances`.

Reply:
Sorry, I mislooked this name. I don't have objections against it.